docs/source/release_notes.rst

```diff
@@ -6,6 +6,43 @@
 Releases
 ========
 
+
+Version 1.3.0
+====================
+
+Notes
+-----
+
+* Update recommended
+
+  * No changes from version 1.3.0-beta.1 release. Collection promoted to certified content with version 1.3.0.
+
+Availability
+------------
+
+* `Automation Hub`_
+* `Galaxy`_
+* `GitHub`_
+
+Reference
+---------
+
+* Supported by IBM z/OS core collection v1.4.0 or later
+* Supported by IBM Z Open Enterprise Python for z/OS 3.9 - 3.12
+* Supported by IBM Z Open Automation Utilities 1.1.1 - 1.2.5
```

Comment:
For ZOAU, shouldn't this say `1.2.2 or later (but prior to 1.3)`. See the beta.1 release section for reference.